packages/edit-site/src/components/navigation-sidebar/navigation-toggle/style.scss

```diff
@@ -6,6 +6,8 @@
 		background: $gray-900;
 		border-radius: 0;
 		display: flex;
+		position: absolute;
```

Comment:
**Note:**
When the navigation sidebar is closed, the skeleton body (i.e. the block editor) no longer spans the entire width of the viewport. The sidebar (and toggle) occupy space at the left edge of the screen. To address this, we position the navigation toggle absolutely when the sidebar is closed.